core/src/style/textStyle.cpp

```diff
@@ -67,12 +70,15 @@ Parameters TextStyle::parseRule(const DrawRule& _rule) const {
     _rule.get(StyleParamKey::transform, transform);
     _rule.get(StyleParamKey::visible, p.visible);
     _rule.get(StyleParamKey::priority, p.priority);
+    if (_rule.get(StyleParamKey::text_source, p.textSource.text)) {
+        p.textSource.isFunction = _rule.isJSFunction(StyleParamKey::text_source);
+    }
 
-    if (transform == "capitalize") {
+    if (transform == capitalize) {
         p.transform = TextTransform::capitalize;
-    } else if (transform == "lowercase") {
+    } else if (transform == lowercase) {
         p.transform = TextTransform::lowercase;
-    } else if (transform == "uppercase") {
+    } else if (transform == uppercase) {
         p.transform = TextTransform::uppercase;
     }
```

Comment:
Yes, it make sense to move this to StyleParam::parseString.

For this PR however, can we get away with the static strings and directly compare with "capitalized", etc.
We can have a separate small PR for moving string-to-enum conversion to StyleParam::parseString.